projects/step-frontend/src/lib/modules/timeseries/time-series.config.ts

```diff
@@ -71,6 +71,45 @@ export class TimeSeriesConfig {
     { type: RangeSelectionType.RELATIVE, relativeSelection: { label: 'Last 3 hours', timeInMs: this.ONE_HOUR_MS * 3 } },
     { type: RangeSelectionType.FULL },
   ];
+
+  static readonly AXES_FORMATTING_FUNCTIONS = {
+    bigNumber: (num: number) => {
+      if (num && num < 10) {
+        return num.toString();
+      }
+      const lookup = [
+        { value: 1, symbol: '' },
+        { value: 1e3, symbol: 'k' },
+        { value: 1e6, symbol: 'M' },
+        { value: 1e9, symbol: 'B' },
+      ];
+      const rx = /\.0+$|(\.[0-9]*[1-9])0+$/;
```

Comment:
Always try to select useful function names and avoid abreviations when possible. From the RegEx I would guess this matches numbers ending in 0 after the decimal point, if so I would name it "matchDecimalEndingZero" or something in that direction.

Ideally the lookup variable would also be a bit more self explaining.